src/monitor.cc

```diff
@@ -271,67 +299,57 @@ void NodeMonitor::setStatistics() {
 
     // milliseconds
     long timeDelta = (cur_time.tv_sec * 1000 + cur_time.tv_usec / 1000)
-        - (instance_->stats_.lastTime_.tv_sec * 1000 + instance_->stats_.lastTime_.tv_usec / 1000);
+        - (stats_.lastTime_.tv_sec * 1000 + stats_.lastTime_.tv_usec / 1000);
 
     // Update the number of requests processed
     // and the ratio CPU/req.
-    instance_->stats_.lastRequests_ = currReqs;
-    instance_->stats_.lastCpuPerReq_ = (reqDelta <= 0) ? 0 : (scpu + ucpu) / reqDelta;
-    instance_->stats_.lastJiffiesPerReq_ = (reqDelta <= 0) ? 0
+    stats_.lastRequests_ = currReqs;
+    stats_.lastCpuPerReq_ = (reqDelta <= 0) ? 0 : (scpu + ucpu) / reqDelta;
+    stats_.lastJiffiesPerReq_ = (reqDelta <= 0) ? 0
         : ((float) (sticks + uticks)) / reqDelta;
 
     // Request delta - requests since last check.
-    instance_->stats_.lastReqDelta_ = reqDelta;
-    instance_->stats_.timeDelta_ = timeDelta;
+    stats_.lastReqDelta_ = reqDelta;
+    stats_.timeDelta_ = timeDelta;
 
     // Update time
-    instance_->stats_.lastTime_.tv_sec = cur_time.tv_sec;
-    instance_->stats_.lastTime_.tv_usec = cur_time.tv_usec;
+    stats_.lastTime_.tv_sec = cur_time.tv_sec;
+    stats_.lastTime_.tv_usec = cur_time.tv_usec;
 
     // Last RPS
-    instance_->stats_.lastRPS_ = (int) (reqDelta / (((double) timeDelta) / 1000));
+    stats_.lastRPS_ = (int) (reqDelta / (((double) timeDelta) / 1000));
 
     // Get currently open requests
-    instance_->stats_.currentOpenReqs_ = getIntFunction("getRequestCount");
+    stats_.currentOpenReqs_ = getIntFunction("getRequestCount");
```

Comment:
These are calls into Javascript, because the values they're obtaining are set by listening on Javascript events emitted by the `Server` object (for the most part).  They are somewhat expensive, but I don't think it's in the scope of this pull request to change this mechanism (which is long-standing).